pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -35,16 +35,12 @@ type OperatorVersionSpec struct {
 	Parameters []Parameter `json:"parameters,omitempty"`
 
 	// Plans maps a plan name to a plan.
-	// +nullable
```

Comment:
Turns out, this annotation leads to [this error](https://github.com/kubernetes/kubernetes/issues/85127) in API server logs